### PR TITLE
[#57] Add wiki content for the available configs when setting up a new project from init script

### PR DESCRIPTION
**Pull request by markgravity**
https://github.com/nimblehq/flutter_templates/issues/57

## What happened 👀

- Add `Setup-More-Available-Configs.md` with more available configs for setup script
- Update `Readme.md` with a link to `Setup-More-Available-Configs.md`

## Insight 📝

n/a

## Proof Of Work 📹

n/a

### CODE REVIEW

File: .github/wiki/Setup-More-Available-Configs.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# More available configs
+| Parameter name | Is mandatory | Description                                                                     |
+| :------------- | :----------: | :------------------------------------------------------------------------------ |
+| APP_VERSION    |      No      | The app version that is set when initialize the project. Default is `0.1.0`     |
+| BUILD_NUMBER   |      No      | The build number that is set when initialize the project. Default is `1`        |
+| json_serializable.field_rename   |      No      | Defines the automatic naming strategy when converting class field names into JSON map keys. <br/>Available options: <br/> - `none`: Use the field name without changes.<br/> - `kebab`: Encodes a field named `kebabCase` with a JSON key `kebab-case`.<br/> - `snake`: Encodes a field named `snakeCase` with a JSON key `snake_case`.<br/> - `pascal`: Encodes a field named `pascalCase` with a JSON key `PascalCase`.        |
```
Comment (doannimble) [inline .github/wiki/Setup-More-Available-Configs.md:6]: @markgravity Should it be `Use the field names without changing`? 🤔

Comment (markgravity) [reply .github/wiki/Setup-More-Available-Configs.md:6]: @doannimble I just copy & paste from their document 😆 
Screen Shot 2022-06-03 at 14 37 05

**Review by manh-t — COMMENTED** [review]
sorry for late 🙇 rest LGTM 👏

File: README.md [hunk 2]
```diff
@@ -27,8 +27,8 @@ Clone the repository
   | PACKAGE_NAME   |     Yes      | The application package name. The naming convention follows `com.your.package`  |
   | PROJECT_NAME   |     Yes      | The application project name. The naming convention follows `your_project_name` |
   | APP_NAME       |     Yes      | The application name.                                                           |
-  | APP_VERSION    |      No      | The app version that is set when initialize the project. Default is `0.1.0`     |
-  | BUILD_NUMBER   |      No      | The build number that is set when initialize the project. Default is `1`        |
+
+  More available configs [here](https://github.com/nimblehq/flutter_templates/wiki/Setup-More-Available-Configs)
```
Comment (manh-t) [inline README.md:31]:
```suggestion
  More available configs [here](https://github.com/nimblehq/flutter_templates/wiki/Generating-A-Project)
```

**Review by doannimble — APPROVED** [review]
LGTM 🚀

**Review by manh-t — COMMENTED** [review]
rest LGTM 👏

**Review by manh-t — APPROVED** [review]
LGTM 🚀

**Pull request merged**